Corona-Warn-App/src/main/res/layout/contact_diary_edit_locations_fragment.xml

```diff
@@ -16,7 +16,9 @@
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent"
             app:navigationIcon="@drawable/ic_close"
-            app:title="@string/contact_diary_edit_locations_title" />
+            app:navigationContentDescription="@string/accessibility_close"
+            app:title="@string/contact_diary_edit_locations_title"
+            android:contentDescription="@string/contact_diary_edit_locations_title"/>
```

Comment:
Please format? Normally `android:` attributes are before `app:`

Reply:
If we want that we should do that automatically/ check with lint.

It's not done anywhere in the app.